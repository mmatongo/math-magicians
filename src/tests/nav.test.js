import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from '../components/Nav';

it('Navbar renders the UI as expected', () => {
  const tree = renderer.create(
    <Router>
      <Nav />
    </Router>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
