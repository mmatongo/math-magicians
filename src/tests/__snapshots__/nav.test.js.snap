// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders the UI as expected 1`] = `
<div
  className="nav-component"
>
  <h1>
    Math Magicians
  </h1>
  <nav
    className="nav"
  >
    <ul
      className="nav-list"
    >
      <li
        className="nav-items"
      >
        <a
          aria-current="page"
          className="active-link"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Home
        </a>
        <span
          className="divider"
        >
           | 
        </span>
      </li>
      <li
        className="nav-items"
      >
        <a
          aria-current={null}
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
        <span
          className="divider"
        >
           | 
        </span>
      </li>
      <li
        className="nav-items"
      >
        <a
          aria-current={null}
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
        <span
          className="divider"
        >
           | 
        </span>
      </li>
    </ul>
  </nav>
</div>
`;
